### ISSUE TITLE
False negative with NamedTuple with wrong typename

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#9514]
```py
from typing import NamedTuple

A = NamedTuple('B', [('a', int)])  # no error
```
`Success: no issues found in 1 source file`

**Comment by gvanrossum** [comment]
This one seems simple to fix -- do you want to give it a try?

**Comment by KotlinIsland** [comment]
Sure :), where should I start?

**Comment by gvanrossum** [comment]
Grep the source code in the mypy subdirectory for NamedTuple to find the code where NamedTuple is handled.

**Comment by momohatt** [comment]
I think this is a duplicate of #4589
